dist/stat_data.py

```diff
@@ -282,8 +282,13 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_lookaside_insert', 'cache overflow table insert calls'),
     CacheStat('cache_lookaside_ondisk', 'cache overflow table on-disk size', 'no_clear,no_scale,size'),
     CacheStat('cache_lookaside_ondisk_max', 'cache overflow table max on-disk size', 'no_clear,no_scale,size'),
+    CacheStat('cache_lookaside_read', 'cache overflow table update reads'),
+    CacheStat('cache_lookaside_read_miss', 'cache overflow table update reads missed'),
+    CacheStat('cache_lookaside_read_squash', 'cache overflow table update reads requiring modify squashing'),
+    CacheStat('cache_lookaside_read_wasted', 'cache overflow table update reads wasted'),
     CacheStat('cache_lookaside_remove', 'cache overflow table remove calls'),
     CacheStat('cache_lookaside_score', 'cache overflow score', 'no_clear,no_scale'),
+    CacheStat('cache_lookaside_write_squash', 'cache overflow table update writes requiring modify squashing'),
```

Comment:
This is tracking how often we have to squash modifies on lookaside eviction to avoid displacing each other in the case where they have the same timestamp and transaction id.